recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "pluggy" %}
-{% set version = "0.5.2" %}
-{% set sha256 = "bd60171dbb250fdebafad46ed16d97065369da40568ae948ef7117eee8536e94" %}
+{% set version = "0.3.1" %}
```

Comment:
Are you OK with this downgrade @nicoddemus ?

Reply:
Yep no problem, please go ahead!